app/src/docs/v1.api-spec.yaml

```diff
@@ -438,6 +444,47 @@ components:
           type: string
           description: 'Contains message about the success, normally "Written successfully"'
           example: Written successfully
+    Problem:
+      required:
+        - type
+        - title
+        - status
+        - detail
+      properties:
```

Comment:
Properties should be alphabetically ordered

Reply:
again, we will be cleaning up the yaml.